Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/WOfSImputation.ipynb

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
<p>are you comparing this to the dea-maps version? could be worth adding a link here so we can see your thinking..</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Yep. So I'll link to [this](http://maps.dea.ga.gov.au/#share=s-iXbqqnVDzDuMnG79VgLH6woQYQQ).</p>

Reply:
Nice.